src/components/Pages/Home.js

```diff
@@ -26,11 +26,12 @@ const Home = ({ topServices, image, intl }) => (
       title={intl.formatMessage(i18n3.checkOutServices)}
       tiles={topServices}
       locale={intl.locale}
-      extraClasses={'floating'}
+      extraClasses={'coa-TopServicesHome'}
```

Comment:
The home page has its own class, here we just make sure it is passed to the PageBanner component in the Home page.